core/modules/loader/CentralClient.h

```diff
@@ -162,14 +164,15 @@ class CentralClient : public Central {
     const int         _defWorkerPortUdp; ///< Default worker UDP port
 
 
-    int _doListMaxLookups; ///< Maximum number of concurrent lookups in DoList  DM-16555
-    int _doListMaxInserts; ///< Maximum number of concurrent inserts in DoList  DM-16555
+    size_t _doListMaxLookups; ///< Maximum number of concurrent lookups in DoList  DM-16555 &&&
+    size_t _doListMaxInserts; ///< Maximum number of concurrent inserts in DoList  DM-16555 &&&
+    int _maxRequestSleepTime{100000}; ///< Time to sleep between checking requests when at max length &&& add config file entry
 
     std::map<std::string, KeyInsertReqOneShot::Ptr> _waitingKeyInsertMap;
-    std::mutex _waitingKeyInsertMtx; ///< protects _waitingKeyInsertMap
+    std::mutex _waitingKeyInsertMtx; ///< protects _waitingKeyInsertMap, _doListMaxInserts
 
-    std::map<std::string, KeyInfoReqOneShot::Ptr> _waitingKeyInfoMap;
-    std::mutex _waitingKeyInfoMtx; ///< protects _waitingKeyInfoMap
+    std::map<std::string, KeyInfoReqOneShot::Ptr> _waitingKeyInfoMap; // &&& change all references of keyInfo to keyLookup &&&, including protobuf keyInfo should only apply to  worker key count and worker key range.
```

Comment:
This line looks way to long. I think we have a limit of 105 characters, or something like this. You may also consider putting this comment in front of the member variable and document it using the Doxygen style:
```
/// &&& change all references of keyInfo to keyLookup &&&, including protobuf keyInfo
/// should only apply to  worker key count and worker key range.
std::map<std::string, KeyInfoReqOneShot::Ptr> _waitingKeyInfoMap;
```